libraries/SRV_Channel/SRV_Channels.cpp

```diff
@@ -213,6 +213,14 @@ const AP_Param::GroupInfo SRV_Channels::var_info[] = {
     // @User: Advanced
     AP_GROUPINFO("_DSHOT_ESC",  24, SRV_Channels, dshot_esc_type, 0),
 
+    // @Param: _GPIO_MASK
+    // @DisplayName: Servo GPIO mask
+    // @Description: This sets a bitmask of outputs which will be available as GPIOs. Any auxillary output with either the function set to -1 or with the corresponding bit set in this mask will be available for use as a GPIO pin
```

Comment:
As NathanE says, we can remove the "This sets a" part of the description. This is what all parameters do so no need to specify this.